.github/workflows/format.yml

```diff
@@ -0,0 +1,39 @@
+name: 'Format Code'
+
+on:
+  push:
```

Comment:
can we make this takeover the pull-request biome action as well?

https://github.com/astrolicious/studiocms/blob/7092e69d0eb1e4a4477444532117af9eab9736ed/.github/workflows/biome-linting.yml#L3-L20

Reply:
also probably